### ISSUE TITLE
Plan chokes on trailing comments in config

### ISSUE THREAD

**Issue by rymiel** [plan-player-analytics/Plan#2297]
### Describe the issue

When using trailing comments in the config, the custom YAML parser being used by Plan gets strangely confused and chops off half of the line.

### Plugin versions

5.4 build 1583

### Additional information

This would probably be fixed by fixing #1363 ! (this issue might even be considered a duplicate of that)

**Comment by AuroraLS3** [comment]
Could you give an example so it's easier to reproduce

**Comment by AuroraLS3** [comment]
Yep the current system supports comments on top of the items but not on the same line
